zk/tests/test_zk.py

```diff
@@ -0,0 +1,96 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import os
+import sys
+import time
+from distutils.version import LooseVersion  # pylint: disable=E0611,E0401
+import pytest
+import subprocess
+import requests
+
+# project
+from datadog_checks.zk import ZookeeperCheck
+import common
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture
```

Comment:
let's add `scope="session"` to avoid spinning up and down the service all the times